src/CoreWCF.Primitives/src/CoreWCF/Security/ISspiNegotiation.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Security.Authentication.ExtendedProtection;
+
+namespace CoreWCF.Security
+{
+
+    internal interface ISspiNegotiation : IDisposable
+  {
+   // DateTime ExpirationTimeUtc { get; }
+
```

Comment:
Do we have any real information behind this property with this implementation? If so, it might be useful to keep this property for debug purposes